### DISCUSSION TITLE
Function calling with Llama?

### DISCUSSION THREAD

**Discussion by Johnhersh** [microsoft/semantic-kernel#4341] [Q&A] [answered]
Is it possible to use SK to do function calling with a local LLM? The examples show how to do it with OpenAI, but I don't see how to apply `ToolCallBehavior.AutoInvokeKernelFunctions` to a local service.

**Comment by stephentoub** [comment, marked as answer]
https://github.com/SciSharp/LLamaSharp/issues/291#issuecomment-1852564173
